hw/ip/hmac/data/hmac.hjson

```diff
@@ -4,6 +4,14 @@
 {
   name:               "hmac",
   one_line_desc:      "Accelerator for SHA-256-based keyed hash message authentication code and the SHA-256 hash function",
+  one_paragraph_desc: '''
+  The OpenTitan HMAC hardware block is a keyed hash based message authentication code generator using [SHA-256][nist-fips-180-4] to check the integrity of an incoming message and a signature signed with the same secret key.
```

Comment:
Adding "optionally" to indicate that this can be disabled SGTM. I don't think we should include more details about this in a one-paragraph description